.vscode/extensions.json

```diff
@@ -2,6 +2,14 @@
   "recommendations": [
     "editorconfig.editorconfig",
     "redhat.vscode-yaml",
-    "biomejs.biome"
+    "biomejs.biome",
+    "github.remotehub",
```

Comment:
Considering this is self described as a proof of concept and is archived, I'd say we remove this too